### PR TITLE
Add a quick check for MST Store Python install

### PR DESCRIPTION
**Pull request by arogl**
## Description

Quick fix for #5467.

Checks if the path for python is under the windows store folder then error and point the user to the beets [documentation](https://beets.readthedocs.io/en/stable/guides/main.html).

Happy for feedback to improve, but thought it best to exit as early as possible.

## To Do

- [x] Documentation.
- [x] Changelog.
- [x] Tests.

### CODE REVIEW

File: beets/ui/__init__.py [hunk 1]
```diff
@@ -1858,6 +1858,16 @@ def main(args=None):
...
     exception handlers that print friendly error messages.
     """
+    if "AppData\\Local\\Microsoft\\WindowsApps" in sys.exec_prefix:
+        log.error(
+            "error: beets is unable to use the Microsoft Store version of "
+            "Python. Please install Python from python.org."
+        )
+        log.error(
+            "error: More details can be found here "
+            "https://beets.readthedocs.io/en/stable/guides/main.html"
```
Comment (snejus) [inline beets/ui/__init__.py:1868]: Consider using a single `log.error` call:

```suggestion
            "error: beets is unable to use the Microsoft Store version of "
            "Python. Please install Python from python.org. See "
            "https://beets.readthedocs.io/en/stable/guides/main.html."
```

**Review by snejus — CHANGES_REQUESTED** [review]

File: beets/ui/__init__.py (hunk 1 above)
Comment (arogl) [reply beets/ui/__init__.py]: Updated

**Comment by snejus** [comment]
Apologies for not addressing this earlier - all looks good, just needs a rebase on latest master.

**Review by snejus — APPROVED** [review]

**Comment by arogl** [comment]
> Apologies for not addressing this earlier - all looks good, just needs a rebase on latest master.

I believe I have managed a rebase 😃

**Pull request merged**